core/source/Math.mint

```diff
@@ -1,5 +1,14 @@
 /* Mathematical functions. */
 module Math {
+  const E = `Math.E`
+  const LN10 = `Math.LN10`
+  const LN2 = `Math.LN2`
+  const LOG10E = `Math.LOG10E`
+  const LOG2E = `Math.LOG2E`
+  const PI = `Math.PI`
+  const SQRT12 = `Math.SQRT1_2`
+  const SQRT2 = `Math.SQRT2`
```

Comment:
Would be nice if these constants have comments as well.

Reply:
Added 👍